test/cmdlineTests/metadata_append_standard_json/input.json

```diff
@@ -0,0 +1,33 @@
+{
+	"language": "Solidity",
+	"sources":
+	{
+		"A":
+		{
+			"content": "// SPDX-License-Identifier: GPL-3.0\npragma solidity >=0.0;\n\ncontract test {
+					struct S {
+						uint x;
+					}
+					S s;
+					function f(bool b) public {
+						s.x |= b ? 1 : 2;
+						assert(s.x > 0);
+					}
+				}"
+		}
+	},
+	"settings":
+	{
+		"metadata":
+		{
+			"append": false
```

Comment:
This is not really a good test because the testing framework is removing `object":"<BYTECODE REMOVED>"`. But it actually removed the bytecode at the end if you test it locally `./solc --standard-json ../../test/cmdlineTests/metadata_append_standard_json/input.json`.

```
 ./solc --standard-json ../../test/cmdlineTests/metadata_append_standard_json/input.json                                 no-append-metadata ✱ ◼
{"contracts":{"A":{"test":{"evm":{"bytecode":{"functionDebugData":{},"generatedSources":[],"linkReferences":{},"object":"608060405234801561001057600080fd5b50610120806100206000396000f3fe608060405234801561001057600080fd5b506004361061002b5760003560e01c806398c3a6c114610030575b600080fd5b61004a600480360381019061004591906100c4565b61004c565b005b8061005857600261005b565b60015b60ff1660008001600082825417925050819055506000806000015411610084576100836100f1565b5b50565b600080fd5b60008115159050919050565b6100a18161008c565b81146100ac57600080fd5b50565b6000813590506100be81610098565b92915050565b6000602082840312156100da576100d9610087565b5b60006100e8848285016100af565b91505092915050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052600160045260246000fd"
....
```
let me know if there is a better way to write a standard json test for this.

Reply:
Funnily enough, if I add the extra args `--pretty-json --json-indent 4`, the bytecode ends up showing. Which solves the issue, actually.

Reply:
I think it's because the regex assumes no space after `:`. It's fine in this case since there's no metadata but we'll have to fix that because in other cases metadata is problematic. So in the long term I would not count on it staying this way and a comment would still be useful.

Or I guess, we could just start using `append: false` in tests :)